openqa-schedule-mm-ping-test

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eux -o pipefail
+
+openqa_url=${openqa_url:-https://openqa.opensuse.org}
+distri=${distri:-opensuse}
+flavor=${flavor:-DVD}
+arch=${arch:-x86_64}
+version=${version:-Tumbleweed}
+test_name=${test_name:-ping_client}
+
+tmpfile=$(mktemp)
+cat > "$tmpfile" << EOF
+---
+products:
+  mm-ping-test:
+    distri: $distri
+    flavor: $flavor
+    arch: $arch
+    version: $version
+
+machines:
+  64bit:
+    backend: qemu
+    settings:
+      WORKER_CLASS: qemu_x86_64,tap
+
+.common: &common
+  BOOT_HDD_IMAGE: '1'
+  DESKTOP: textmode
+  IS_MM_SERVER: '1'
+  NICTYPE: tap
+  QEMU_DISABLE_SNAPSHOTS: '1'
+  YAML_SCHEDULE: schedule/functional/mm_ping.yaml
+
+job_templates:
+  ping_server:
+    product: mm-ping-test
+    machine: 64bit
+    settings:
+      <<: *common
+      HOSTNAME: server
+  ping_client:
+    product: mm-ping-test
+    machine: 64bit
+    settings:
+      <<: *common
+      HOSTNAME: client
+      PARALLEL_WITH: ping_server
+EOF
+
+openqa-cli schedule \
+    --monitor \
+    --host "$openqa_url" \
+    --param-file SCENARIO_DEFINITIONS_YAML="$tmpfile" \
+    DISTRI="$distri" VERSION="$version" FLAVOR="$flavor" ARCH="$arch" \
+    BUILD="$(date +%+4Y%m%d-%H%M)" _GROUP_ID=0 \
```

Comment:
How about `date -Im`

Reply:
I was trying to not use `:` in the build as that is sometimes used as a delimiter. But I am ok with any sensible format, if you prefer ISO 8601 a will make that change.

Reply:
Yes, I like ISO 8601 :)

Reply:
Note that `date +%+4Y%m%d-%H%M` is *almost* an ISO 8601 timestamp. The only validation I see is the dash (the date and time need to be separated using a `T` instead). An ISO 8601 timestamp does not need to contain any colons as you can omit most separators (see https://en.wikipedia.org/wiki/ISO_8601).